src/menu.c

```diff
@@ -1688,12 +1686,12 @@ static  int get_graph_pcg(void)
 static  void    cb_graph_pcg(Q8tkWidget *widget, void *p)
 {
     if (widget) {
-    use_pcg = (int)p;
-    memory_set_font();
+        use_pcg = (intptr_t)p;
+        memory_set_font();
     }
 
-    if (use_pcg) { q8tk_widget_set_sensitive(graph_font_widget, FALSE); }
-    else         { q8tk_widget_set_sensitive(graph_font_widget, TRUE);  }
+    if (use_pcg)  { q8tk_widget_set_sensitive(graph_font_widget, FALSE);  }
```

Comment:
```suggestion
    if (use_pcg) { q8tk_widget_set_sensitive(graph_font_widget, FALSE); }
```
Extra spaces don't seem necessary.